sites/skeleton.dev/src/routes/+layout.svelte

```diff
@@ -187,7 +176,14 @@
 </svelte:head>
 
 <!-- Overlays -->
-<Modal components={modalComponentRegistry} />
```

Comment:
Stuff "has" to be moved inline - otherwise you don't get the convenience of generics.

Reply:
I don't particularly mind inlining props like this, but @endigo9740 may find this sacrilegious 😄. I'm fine with either.